extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -1420,7 +1432,7 @@ public Void apply(@Nullable List<Void> input)
           {
             return null;
           }
-        }, workerExec
+        }
```

Comment:
Minor comment: this code would be more readable if this pattern was expressed in a function like `ListenableFuture<Void> asVoidFuture(ListenableFuture<?> future)`